local_vars/local_vars.cc

```diff
@@ -228,6 +228,14 @@ class LocalNameInserter {
             return tree;
         }
 
+        size_t i = scopeStack.size() - 1;
+        for (; i >= 0; i--) {
```

Comment:
This loop condition will always be true if `i` has type `size_t`. 😅

Reply:
ah thanks